diff --git a/system/database/DB_active_rec.php b/system/database/DB_active_rec.php
index 10febb1..fd8069a 100755
--- a/system/database/DB_active_rec.php
+++ b/system/database/DB_active_rec.php
@@ -2039,7 +2039,41 @@ class CI_DB_active_record extends CI_DB_driver {
 
 		$this->_reset_run($ar_reset_items);
 	}
+
+  /**
+   * BEGIN SCIGIT PATCHES
+   */
+  /**
+   * Get SELECT query string
+   *
+   * Compiles a SELECT query string and returns the sql.
+   *
+   * @access  public
+   * @param   string  the table name to select from (optional)
+   * @param   boolean TRUE: resets AR values; FALSE: leave AR vaules alone
+   * @return  string
+   */
+  public function get_compiled_select($table = '', $reset = TRUE)
+  {
+      if ($table != '')
+      {
+          $this->_track_aliases($table);
+          $this->from($table);
+      }
+
+      $select =  $this->_compile_select();
+
+      if ($reset === TRUE)
+      {
+          $this->_reset_select();
+      }
+
+      return $select;
+  }
+  /**
+   * END SCIGIT PATCHES
+   */
 }
 
 /* End of file DB_active_rec.php */
-/* Location: ./system/database/DB_active_rec.php */
\ No newline at end of file
+/* Location: ./system/database/DB_active_rec.php */
